ch6/parse.zig

```diff
@@ -0,0 +1,11 @@
+const std = @import("std");
+
+pub fn main() void {
+    const result = std.fmt.parseFloat(f64,"3.1415") catch |err| {
+        std.debug.print("Failed to parse float: {s}\n", .{@errorName(err)});
+
+        return;
+    };
+
+    std.debug.print("Parsed float: {}\n", .{result});
+}
```

Comment:
is the old custom parseFloat function be available as well?

Reply:
The function could be called because it's std, but the execution scope is done after `result` is defined.